extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -249,6 +251,9 @@ export class AmpConsent extends AMP.BaseElement {
    * Listen to external consent flow iframe's response
    */
   enableExternalInteractions_() {
+    // Should we also listen for consentType being passed in?
+    // i.e. data['type']? Then use this inconjunction with
+    // consent string to update state manager?
```

Comment:
@zhouyx

Reply:
Didn't realize this is a question to me. Yes as you summarize we should definitely support his. consentStringType and consentString are always set/stored/retrieved together.